src/Compilers/CSharp/Portable/Symbols/Source/SourceMemberFieldSymbol.cs

```diff
@@ -323,7 +323,7 @@ public sealed override bool HasInitializer
             get { return _hasInitializer; }
         }
 
-        public VariableDeclaratorSyntax VariableDeclaratorNode
+        private VariableDeclaratorSyntax VariableDeclaratorNode
```

Comment:
I am talking about the contract between this class and SourceFixedFieldSymbol, which you made implicit by replacing the use of this property with a type cast.

---
In reply to: [80959907](https://github.com/dotnet/roslyn/pull/13999#discussion_r80959907) [](ancestors = 80959907,80850539,80816186,80815616)